### PR TITLE
Fixes #YUPTOO-48 - Integrate Yuptoo with Payload tracker

### PR DESCRIPTION
**Pull request by patilsuraj767**

### CODE REVIEW

File: yuptoo/processor/utils.py [hunk 1]
```diff
@@ -58,6 +59,22 @@ def download_report(consumed_message):
...
+        "org_id": request_obj.get("org_id"),
+        "request_id": request_obj.get("request_id"),
+        "payload_id": request_obj.get("request_id"),
+        "service": "yuptoo",
+        "status": status,
+        "status_msg": status_msg,
+        "date": datetime.datetime.now().isoformat(),
+    }
+
+    return message
```
Comment (kgaikwad) [inline yuptoo/processor/utils.py:75]:
```suggestion
def tracker_message(request_obj, status, status_msg):
    return {
        "account": request_obj.get("account"),
        "org_id": request_obj.get("org_id"),
        "request_id": request_obj.get("request_id"),
        "payload_id": request_obj.get("request_id"),
        "service": "yuptoo",
        "status": status,
        "status_msg": status_msg,
        "date": datetime.datetime.now().isoformat(),
    }

     
```

**Review by kgaikwad — APPROVED** [review]
@patilsuraj767  - Looks good to me.
Added one nitpick suggestion.

File: yuptoo/processor/utils.py (hunk 1 above)
Comment (kgaikwad) [reply yuptoo/processor/utils.py:75]: @patilsuraj767 - one more thing about date value do we want to consider `utcnow()` or `now()`?

Comment (patilsuraj767) [reply yuptoo/processor/utils.py]: I have copied it as is from puptoo.

**Review by kgaikwad — APPROVED** [review]

**Pull request merged**